tool/generator/lib/src/projection/utils.dart

```diff
@@ -1,7 +1,6 @@
 // Useful utilities
 
-import 'dart:io';
-
+import 'package:path/path.dart' as p;
```

Comment:
Can we make this `path` rather than `p`? Might be more readable? I think that may be more idiomatic (same way that most people import `math/math.dart` as `math`...

Reply:
Done